python/py-parameterized/Portfile

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-parameterized
+version             0.7.0
+revision            0
+
+categories-append   testing
+platforms           darwin
+supported_archs     noarch
+license             FreeBSD
+maintainers         nomaintainer
+
+description         Parameterized testing with any Python test framework
+long_description    ${description}
+
+homepage            https://github.com/wolever/parameterized
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+distname            ${python.rootname}-${version}
+
+checksums           sha256  d8c8837fb677ed2d5a93b9e2308ce0da3aeb58cf513120d501e0b7af14da78d5 \
+                    rmd160  03f016fca3efcaa1d556625dab260fc3754d759a \
+                    size    20751
+
+python.versions     37
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
+
+    # there are five different ways of running tests, see tox.ini
+    # this runs just one
+    test.run        yes
```

Comment:
in ```test.py``` it does ```from nose.tools import assert_equal, assert_raises``` so even if you run the tests with ```unittest``` you still need ```nose``` installed. The package doesn't declare this, so there isn't much ```upt``` can do about this...

Reply:
It declares it in a super strange way. The strange thing is that it worked in trace mode without any additional dependencies declared, so I'm now confused why.

Reply:
Thank you. I was also missing `mock`.